### DISCUSSION TITLE
Optionally disable / hide the panel menu in dashboards

### DISCUSSION THREAD

**Discussion by ryantxu** [grafana/grafana#38221] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

Similar to dashboard editable flag, it would be nice to disable the panel menu:
![image](https://user-images.githubusercontent.com/705951/107830017-d5aa3100-6d3f-11eb-8b85-319a301ca7d0.png)

![image](https://user-images.githubusercontent.com/705951/107830099-fbcfd100-6d3f-11eb-8297-fb8e5bc668a4.png)

**Why is this needed**:
This options can be a distraction for operational dashboards

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by GarbatyGrabarz** [comment]
Is there any progress on this request?

**Comment by StuCurry** [comment]
+1，Is there any progress on this request? we need it.

**Comment by Maximus48p** [comment]
Is there any progress on this request?

**Comment by wtquin** [comment]
This would be very helpful as it'd allow you to keep your custom panels' configuration saved but invisible when you don't need it at the moment. Right now the only option would be keeping the config file's contents somewhere else for safekeeping and manually pasting the panel back in when needed (or commenting it out), but this begs for a graphical option to manage.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
